src/app/lockers/scanner/page.tsx

```diff
@@ -0,0 +1,80 @@
+'use client'
+import styles from './page.module.scss'
+import CameraFeed from '@/app/components/Camera/CameraFeed'
+import type { CameraState } from '@/app/components/Camera/Types'
+import Button from '@/app/components/UI/Button'
+import { useRouter } from 'next/navigation'
+import jsQR from 'jsqr'
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+export default function Scanner() {
+    const videoRef = useRef<HTMLVideoElement | null>(null)
+    const [cameraState, setCameraState] = useState<CameraState>('Off')
+    const [hasAsked, setHasAsked] = useState<boolean>(false)
+    const router = useRouter()
+
+    const width = 400
```

Comment:
For consistancy write these with capital letters, and maybe move them outside the funcion